src/platform/silabs/wifi/wfx_host_events.h

```diff
@@ -94,34 +95,52 @@ typedef struct __attribute__((__packed__)) sl_wfx_mib_req_s
     sl_wfx_mib_req_body_t body;
 } sl_wfx_mib_req_t;
 
-#else /* End WF200 else RS911x */
+#else /* End WF200 else 916,917 NCP and 917 SoC */
 
 #include "wfx_msgs.h"
 
 #if (SLI_SI91X_MCU_INTERFACE | EXP_BOARD)
```

Comment:
isn't this condition redundant? considering
`SLI_SI91X_MCU_INTERFACE` for SiWx917 SoC,
`EXP_BOARD` is for rs911x and 917 NCP correct?

Reply:
`EXP_BOARD` is only for **917 NCP** and `SLI_SI91X_MCU_INTERFACE` for **SiWx917 SoC**